src/main/python/systemds/operator/nodes/matrix.py

```diff
@@ -343,6 +345,32 @@ def cholesky(self, safe: bool = False) -> 'Matrix':
         """
         return Matrix(self.sds_context, 'cholesky', [self])
 
+    def fft(self):
+        """
+        Performs the Fast Fourier Transform (FFT) on this matrix.
+        Returns a Matrix object representing the FFT (complex numbers represented as two matrices).
```

Comment:
the syntax for documentation is not followed, take a look at some of the other examples.

Reply:
this code is not even necessary. Had forgotten to remove it. Removed it now